custom-movement.js

```diff
@@ -0,0 +1,388 @@
+import { defs, tiny } from "./external/common.js";
```

Comment:
I'm assuming custom-movement.js defines all the key-press movements we talked abt, and is called by wonderland.js which is then called by main-scene.js? just making sure i get the flow correctly

Reply:
Yes!